py/dml/structure.py

```diff
@@ -1987,15 +1989,68 @@ def get_group_registers(group, indexvars):
 
     return regs
 
+def param_linear_int(param):
+    '''If possible, return tuple of coefficients (i0, i1, .., iN, k) such
+    that expr can be equivalently expressed as (i0*v0 + i1*v1 + ... +
```

Comment:
This comment confused me a fair bit, mostly because what 'expr' is is not obvious at point of reading. Consider "expr of param" or similar

Reply:
Good catch! I initially wrote the comment on the inner function, and forgot to update it when moving it to the outer function.